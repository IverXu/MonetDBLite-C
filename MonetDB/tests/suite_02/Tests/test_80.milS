setoid(oid(2000000));
#determine cost components of a join
# load("k10_unique1");

b:=k10_unique1.copy;
printf("#~BeginVariableOutput~#\n"); cpu.print; printf("#~EndVariableOutput~#\n");c:=join(b.reverse,k10_unique1);
printf("#~BeginVariableOutput~#\n"); cpu.print; printf("#~EndVariableOutput~#\n");
d:=join(b.reverse,k10_unique1);
printf("#~BeginVariableOutput~#\n"); cpu.print; printf("#~EndVariableOutput~#\n");
commit;
printf("#~BeginVariableOutput~#\n"); cpu.print; printf("#~EndVariableOutput~#\n");
printf("#~BeginVariableOutput~#\n"); memory.print; printf("#~EndVariableOutput~#\n");
printf("#~BeginVariableOutput~#\n"); io.print; printf("#~EndVariableOutput~#\n");
quit;
