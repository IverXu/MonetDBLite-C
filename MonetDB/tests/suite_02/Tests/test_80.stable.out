stdout of test 'test_80` in directory 'tests/suite_02` itself:

#> Mtest:	milS test_80 milS
#> Mtest:	Mtimeout -timeout 210 Mserver -modulesilent -yaccsilent -d8 -db tests_suite_02 -single test_80.milS
# Monet Database Server V4.1
# Copyright (c) 1993-1999, CWI & SION. All rights reserved.
# compiled for IRIX646.5/32bit on Fri Jan 14 01:04:33 MET 2000.
#script:test_80.milS

loaded;
#---------------------------------#
# module	| usage_count	  #
#---------------------------------#
[ "algebra",	  1		  ]
[ "arith",	  1		  ]
[ "bat",	  1		  ]
[ "kernel",	  1		  ]
[ "str",	  1		  ]
[ "sys",	  1		  ]
[ "trans",	  1		  ]

Try: modules;                - for other modules.
     ls;                     - for persistent bat listing.
     sigs("<module>")        - for available commands per module.
     help("<command>")       - for help one-liner for a command.

#You are system administrator (adm)!
setoid(oid(2000000));
#determine cost components of a join
# load("k10_unique1");

b:=k10_unique1.copy;
printf("#~BeginVariableOutput~#\n"); cpu.print; printf("#~EndVariableOutput~#\n");c:=join(b.reverse,k10_unique1);
#~BeginVariableOutput~#
#-------------------------#
# BAT:		  cpu	  #
# (str)		  (int)	  #
#-------------------------#
[ "elapsed",	  0	  ]
[ "user",	  140	  ]
[ "elapuser",	  0	  ]
[ "system",	  60	  ]
[ "elapsystem",	  0	  ]
#~EndVariableOutput~#
printf("#~BeginVariableOutput~#\n"); cpu.print; printf("#~EndVariableOutput~#\n");
#~BeginVariableOutput~#
#-------------------------#
# BAT:		  cpu	  #
# (str)		  (int)	  #
#-------------------------#
[ "elapsed",	  0	  ]
[ "user",	  150	  ]
[ "elapuser",	  10	  ]
[ "system",	  60	  ]
[ "elapsystem",	  0	  ]
#~EndVariableOutput~#
d:=join(b.reverse,k10_unique1);
printf("#~BeginVariableOutput~#\n"); cpu.print; printf("#~EndVariableOutput~#\n");
#~BeginVariableOutput~#
#-------------------------#
# BAT:		  cpu	  #
# (str)		  (int)	  #
#-------------------------#
[ "elapsed",	  0	  ]
[ "user",	  160	  ]
[ "elapuser",	  10	  ]
[ "system",	  60	  ]
[ "elapsystem",	  0	  ]
#~EndVariableOutput~#
commit;
printf("#~BeginVariableOutput~#\n"); cpu.print; printf("#~EndVariableOutput~#\n");
#~BeginVariableOutput~#
#-------------------------#
# BAT:		  cpu	  #
# (str)		  (int)	  #
#-------------------------#
[ "elapsed",	  0	  ]
[ "user",	  160	  ]
[ "elapuser",	  0	  ]
[ "system",	  70	  ]
[ "elapsystem",	  10	  ]
#~EndVariableOutput~#
printf("#~BeginVariableOutput~#\n"); memory.print; printf("#~EndVariableOutput~#\n");
#~BeginVariableOutput~#
#---------------------------------#
# BAT:		  mem		  #
# (str)		  (int)		  #
#---------------------------------#
[ "memincr",	  1613824	  ]
[ "arena",	  1662960	  ]
[ "ordblks",	  111		  ]
[ "smblks",	  7680		  ]
[ "hblkhd",	  15360		  ]
[ "hblks",	  480		  ]
[ "usmblks",	  309664	  ]
[ "fsmblks",	  442976	  ]
[ "uordblks",	  772232	  ]
[ "fordblks",	  122728	  ]
#~EndVariableOutput~#
printf("#~BeginVariableOutput~#\n"); io.print; printf("#~EndVariableOutput~#\n");
#~BeginVariableOutput~#
#-------------------------#
# BAT:		  io	  #
# (str)		  (int)	  #
#-------------------------#
[ "maxrss",	  1888	  ]
[ "minflt",	  128	  ]
[ "majflt",	  1	  ]
[ "nswap",	  0	  ]
[ "inblock",	  3	  ]
[ "oublock",	  8	  ]
[ "nvcsw",	  19	  ]
[ "ninvcsw",	  8	  ]
#~EndVariableOutput~#
quit;
